### DISCUSSION TITLE
Help me to resolve this issue

### DISCUSSION THREAD

**Discussion by aljumma** [termux/termux-packages#12302] [Q&A]
![Screenshot_20221009_051637](https://user-images.githubusercontent.com/115381216/194733006-ae3f9c61-0b7e-4076-bbd4-c5a8a6ae7688.jpg)

**Comment by sylirre** [comment]
About errors:

`AH00558`: Android doesn't have FQDN and doesn't let user to set a one anyway (if not rooted). In general, you don't need FQDN on Android device. Host name is `localhost` and domain name is `localdomain`, i.e. Linux defaults. That's why Apache complains about that. It's problematic to use virtual hosts in Termux.

`Address already in use`: means that server is already running. You even successfully used `curl` to confirm that. You can't spawn multiple servers on same TCP port. Use `ps` to determine what software is currently running in Termux. Note that all Termux web server packages are configured to use same port 8080 and you can't run them in parallel without changing configuration.

`AH00015`: probably Apache was unable to create log file somewhere under `$PREFIX/var/log/*`. You may need to edit configuration file or manually create a missing subdirectory under `$PREFIX/var/log`.
